### PR TITLE
Journals Week 5

### PR DESCRIPTION
**Pull request by karjac14**

### CODE REVIEW

**Comment by s4dmw** [comment]
good start :+1: 

it's still hard to believe we've only been here 5 weeks...feels like at least 5 months

**Comment by rosalsm** [comment]
very energetic Karlo :+1:

**Comment by drumslayert** [comment]
nice work :+1:  Were half way there bro!

File: resource-week-5.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+
+###Encapsulation with JS!
+
+Resource Link: http://www.intertech.com/Blog/encapsulation-in-javascript/
+_By: Jason Shapiro_
+
+#####Why we encapsulate data?
+
+Encapsulation includes the idea that the data of an object should not be directly exposed.  Instead, callers that want to achieve a given result are coaxed into proper usage by invoking methods (rather than accessing the data directly).
```
Comment (ShayHall) [inline resource-week-5.md]: Make sure your readers know this is a direct quote and not your own summary of the resource.

**Comment by ShayHall** [comment]
I like the thirst for knowledge that you bring to the classroom and your reflective journal shows that. :+1: 
At first glance, that looks like a good resource on encapsulation.  Be sure to site the source when quoting directly. :+1: 
I think finding one thing in SASS to focus on would be more than enough for this tutorial. :+1:

**Pull request merged**